README.md

```diff
@@ -21,14 +21,20 @@ Follow instructions [here](https://git-scm.com/downloads) to install the latest
 
 1. Clone this repository on your computer with the following command: `git clone https://github.com/devcongress/elections-api.git`
 2. Run `bundle install` to grab all the dependencies of the project
-3. Setup your environment variables using the intructions [here](http://railsapps.github.io/rails-environment-variables.html). Essentially you need to have variable values for `ELECTIONS_DB`, `ELECTIONS_DB_TEST`, `ELECTIONS_DB_USERNAME`, `ELECTIONS_DB_PASSWORD`
+<<<<<<< HEAD
+3. Setup your environment variables using the instructions [here](http://railsapps.github.io/rails-environment-variables.html). Essentially you need to have variable values for `ELECTIONS_DB`, `ELECTIONS_DB_TEST`, `ELECTIONS_DB_USERNAME`, `ELECTIONS_DB_PASSWORD`
 4. Run `rails db:create` to create your database
 5. Run `rails db:migrate` to run all migrations
+=======
+3. Create a _.env_ file in the project root directory and add the values for `ELECTIONS_DB`, `ELECTIONS_DB_TEST`, `ELECTIONS_DB_USERNAME`, `ELECTIONS_DB_PASSWORD` variables. All environment variables go into this file.
+4. Run `rake db:create` to create your database
+5. Run `rake db:migrate` to run all migrations
+>>>>>>> env-config
 6. Run `rails server` to start your server
 
 ## How to run the test suite
 
-To run the test suite, run `rails test`
+To run the test suite, run `rake test`
```

Comment:
We are using rails 5 which allows all rake commands to run through the rails command. The recommended  method should be `rails` ref: http://guides.rubyonrails.org/command_line.html & http://blog.bigbinary.com/2016/01/14/rails-5-supports-rake-commands-using-rails.html